extensions/interactions/FractionInput/directives/FractionInputValidationServiceSpec.js

```diff
@@ -190,8 +204,22 @@ describe('FractionInputValidationService', function() {
     }]);
   });
 
-  it('should catch identical rules as redundant', function() {
+  it('should not catch equals followed by equivalent as redundant', function() {
     answerGroups[0].rules = [equalsOneRule, equivalentToOneRule];
```

Comment:
Can you add a check for the reverse condition for this where a warning should appear? I saw one with equivalentToOneAndSimplestFormRule but not any with equivalentToOneRule

Reply:
Done. Thanks 👍